src/wp-includes/class-wp-scripts.php

```diff
@@ -886,10 +939,11 @@ public function add_data( $handle, $key, $value ) {
 	 * @since 6.3.0
 	 * @see WP_Scripts::should_delay_inline_script()
 	 *
+	 * @param string[] $handles Handles to check.
 	 * @return bool True if the inline script present, otherwise false.
 	 */
-	public function has_delayed_inline_script() {
-		foreach ( $this->registered as $handle => $script ) {
+	public function has_delayed_inline_script( array $handles ) {
```

Comment:
We don't generally do type casting in functional parameters. This is only available since PHP 7 which is still above the minimum supported version. Can we avoid this for now?

Reply:
For context: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#type-declarations

Reply:
Well how about that. 👍🏻